README.md

```diff
@@ -1,25 +1,28 @@
-# Vol-E (Volume Explorer)
+# Vol-E App
+
+Volume Explorer (Vol-E) is a browser based volume viewer built with React and WebGL (Three.js). This package wraps the [vole-core](https://github.com/allen-cell-animated/volume-viewer) library.
 
 For the latest stable release, please visit https://allen-cell-animated.github.io/website-3d-cell-viewer-release/
 
-This is a browser based volume viewer built with React and WebGL (Three.js).
 Volume data is provided to the core 3d viewer via one of the following file formats:
-* a url to a OME-ZARR image
-* a url to a OME-TIFF file
-* a json file containing dimensions and other metadata, and texture atlases (png files containing volume slices tiled across the 2d image). These texture atlases must be prepared in advance before loading into this viewer.
 
+- a url to a OME-ZARR image
+- a url to a OME-TIFF file
+- a json file containing dimensions and other metadata, and texture atlases (png files containing volume slices tiled across the 2d image). These texture atlases must be prepared in advance before loading into this viewer.
 
 The volume shader itself is a heavily modified version of one that has distant origins in [Bisque](http://bioimage.ucsb.edu/bisque).
 
 ## to use
+
 - `https://allen-cell-animated.github.io/website-3d-cell-viewer-release/?url=path/to/ZARR`
-- for more url parameters, see `public/index.tsx`
+- for more url parameters, see [`website\utils\url_utils.ts#L82`](website\utils\url_utils.ts#L82)
```

Comment:
This is not really documentation and isn't readable to scientists... I'm going to write something up and open it as a spin-off.

Reply:
Update: Done! This is now #357